tests/NHapi.NUnit/Parser/BadInputTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace NHapi.NUnit.Parser
+{
+    using System;
+    using System.IO;
+
+    using global::NUnit.Framework;
+
+    using NHapi.Base;
+    using NHapi.Base.Parser;
+
+    [TestFixture]
+    public class BadInputTests
+    {
+        private const string AOOR1 = "TestData/BadInputs/aoor1";
+        private const string SYS1 = "TestData/BadInputs/sys1";
+        private const string NULLREF1 = "TestData/BadInputs/nullref1";
+        private const string IOOR1 = "TestData/BadInputs/ioor1";
+        private const string REGEX1 = "TestData/BadInputs/regex1";
+        private const string SYS2 = "TestData/BadInputs/sys2";
+        private const string HANG1 = "TestData/BadInputs/hang1";
+        private const string SO = "TestData/BadInputs/stackoverflow1";
+        private const string SO2 = "TestData/BadInputs/stackoverflow2";
+
+        [Test]
+        [Timeout(2000)]
+        [TestCase(new object[] { AOOR1 })]
```

Comment:
these test case attributes can just look like this:

```csharp
[TestCase(AOOR1)]
...
```

There shouldn't be any need for the object syntax.
Could I ask what the timeout is for?

Reply:
Yes the object syntax doesnt make sense
The timeout is for the hang and stackoverflow inputs, otherwise it never completes the test in the unfixed version